forked-daapd.conf.in

```diff
@@ -246,6 +246,21 @@ audio {
 #	adjust_period_seconds = 100
 }
 
+# multiple Alsa sections for multiple devices
```

Comment:
Can you try to make purpose and use of this configuration more clear? Also be consistent with the spelling (ALSA, not Alsa) and capitalization.